src/coreclr/hosts/unixcoreconsole/CMakeLists.txt

```diff
@@ -31,8 +31,4 @@ if(NOT CLR_CMAKE_PLATFORM_ANDROID)
   )
 endif()
 
-add_dependencies(coreconsole
-    coreclr
```

Comment:
I think so. Just I have one question, is it has any problem in windows environment if we remove this build dependency? It seems to build corerun and coreconsole during cross component build in Windows (and not installed), but I'm not good at Windows environment. If it also have no problem in Windows, I think it's better to merge add_subdirectory(src/coreclr/hosts) with Windows and Linux environment, by excluding add_subdirectory(src/coreclr/hosts) at root/CMakeLists.txt and use if statement using NOT DEFINED CLR_CROSS_COMPONENTS_BUILD instead of WIN32 in src/coreclr/CMakeLists.txt.